packages/web-frontend/src/selectors/projectSelectors.js

```diff
@@ -48,7 +50,7 @@ export const selectActiveTileSetKey = state => state.map.activeTileSetKey;
 export const selectActiveTileSet = createSelector(
   [selectTileSets, selectActiveTileSetKey],
   (tileSets, activeTileSetKey) => {
-    return tileSets.find(tileSet => tileSet.key === activeTileSetKey);
+    return tileSets.find(tileSet => tileSet.key === activeTileSetKey) || tileSets[0];
```

Comment:
When rendering the page, the initial `tileSets` here were an array with only two items(osm and satellite), and I couldn't find where they were set. This will cause error because with new changes, `activeTileSetKey` here is `laosOpenStreets`.